op3_ball_detector/src/ball_detector.cpp

```diff
@@ -138,11 +142,19 @@ void BallDetector::process()
   if (enable_ == false)
     return;
 
-  if (switch_detection_flag_ == true && 50 <= num_call_++)
+  switch_detection_flag_ = true;
```

Comment:
looks like this testing switch got left in (ln 145)

Reply:
Fixed. Also commented code for testing the distribution range. We can remove it once we have tested on the bot and determined it works.